packages/Search/src/DTK_Search.hpp

```diff
@@ -0,0 +1,18 @@
+/****************************************************************************
+ * Copyright (c) 2012-2019 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef DTK_SEARCH_HPP
+#define DTK_SEARCH_HPP
+
+#include <DTK_DistributedSearchTree.hpp>
+#include <DTK_LinearBVH.hpp>
```

Comment:
Do we want to only put these two headers or should we put all of them? I realize that the others should be included by these two but what happens if we change the headers later on? I don't have an opinion, just asking

Reply:
As a rule of thumb, we could add all but `*Details*`.